packages/phpstan-rules/tests/Rules/DeadCode/UnusedPublicClassConstRule/Source/ConstantUser.php

```diff
@@ -4,11 +4,16 @@
 namespace Symplify\PHPStanRules\Tests\Rules\DeadCode\UnusedPublicClassConstRule\Source;
 
 use Symplify\PHPStanRules\Tests\Rules\DeadCode\UnusedPublicClassConstRule\Fixture\SkipUsedPublicConstant;
+use Symplify\PHPStanRules\Tests\Rules\DeadCode\UnusedPublicClassConstRule\Fixture\SkipUsedPublicConstantInSubclass;
 
 final class ConstantUser
 {
     public function run()
     {
         return SkipUsedPublicConstant::USED;
     }
+    public function run2()
+    {
+        return SkipUsedPublicConstantInSubclass::USED;
```

Comment:
when the constant is referenced via a subclass, I think it should not be reported as unused... wdyt?